examples/sycl/00_bmg_gemm/00_bmg_gemm.cpp

```diff
@@ -156,7 +156,7 @@ struct ExampleRunner {
   using ElementAcc = typename Gemm::ElementAccumulator;
 
   using CollectiveEpilogue = typename Gemm::CollectiveEpilogue;
-  using ElementC = typename Gemm::ElementC;
+  using ElementC = typename CollectiveEpilogue::ElementOutput;
```

Comment:
I think this change is wrong. Why change this? Did you check this is correct if dtype of C and output is different?

Reply:
It assums C and D has same dtype here. I also think we need to support more dtype combinations.